.github/workflows/pullpreview.yml

```diff
@@ -55,6 +59,20 @@ jobs:
           done
           echo "$BASIC_AUTH" > .htpasswd
 
+      - name: Download passed artifact.
+        # https://github.com/actions/download-artifact
+        uses: actions/download-artifact@v4
+        if: ${{ inputs.ARTIFACT_NAME }}
```

Comment:
Write the `if` directly below the name to match the style in the other workflows.